openfisca_us/variables/finance/benefit/snap.py

```diff
@@ -3,6 +3,16 @@
 from openfisca_us.tools.general import *
 
 
+class snap_gross_income(Variable):
+    value_type = float
+    entity = Person
+    definition_period = YEAR
+    label = "Gross income for the person"
+
+    def formula(persons, period, parameters):
+        return persons("snap_gross_income", period)
```

Comment:
this recursion won't work. if it's not a formula based on other variables, you can omit the `formula` section.